### PR TITLE
:sparkles: page profile/[email]/graphs

### LINKED ISSUE #142: Be able to view radar and progressbars for another profile than the user or zenika

### PR DESCRIPTION
**Pull request by MailyLehoux**
fix #142

### CODE REVIEW

**Comment by MailyLehoux** [comment]
resolve #241

File: src/components/PageWithSkillList.tsx [hunk 1]
```diff
@@ -253,7 +252,7 @@ const PageWithSkillList = ({
                     <Link href={`/skills/${context}/${category.name}`}>
                       <button
                         className={`${
-                          add && context != "zenika"
+                          add && context === "mine"
```
Comment (benjaminbrgs) [inline src/components/PageWithSkillList.tsx:255]:
```suggestion
                          add 
```

Could be removed as it is now checked line 245

File: src/components/PageWithSkillList.tsx [hunk 2]
```diff
@@ -269,7 +268,7 @@ const PageWithSkillList = ({
                     >
                       <button
                         className={`${
-                          add && context != "zenika"
+                          add && context === "mine"
```
Comment (benjaminbrgs) [inline src/components/PageWithSkillList.tsx:271]:
```suggestion
                          add
```

same

**Review by benjaminbrgs — CHANGES_REQUESTED** [review]
Great feature ! I really like this, it's helpful 👏🏼 

Is there a way to display an error or redirect to the 404 page when the user does not exist ?
Exemple : `http://localhost:3000/profile/fakeemail@zenika.com/graphs`

**Comment by MailyLehoux** [comment]
good catch @benjaminbrgs thanks :100:

**Review by benjaminbrgs — APPROVED** [review]

**Pull request merged**